src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: New Relic Kubernetes components
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Kubernetes components
+metaDescription: "Learn what components are deployed after installing the Kubernetes integration."
+---
+
+import kubernetesv3Integration from 'images/kubernetes_diagram_v3-integration.png'
+
+The New Relic Kubernetes integration gives you full observability into the health and performance of your cluster, whether you're running Kubernetes on-premises or in the cloud. It gives you visibility into Kubernetes namespaces, deployments, replicasets, nodes, pods, and containers.
+
+## Architecture [#architecture]
+
+The `newrelic-infrastructure` DaemonSet is the main component of the integration. It's divided in three different components: `nrk8s-ksm`, `nrk8s-kubelet`, and `nrk8s-controlplane`. The first is a deployment and the next two are DaemonSets.
+
+Each of the components has two containers:
+
+1. A container for the integration, responsible for collecting metrics.
+2. A container with the New Relic infrastructure agent, which is used to send the metrics to New Relic.
+
+<img
+  title="Diagram showing the integration installed in a 3 nodes cluster"
+  alt="Diagram showing the integration installed in a 3 nodes cluster"
+  src={kubernetesv3Integration}
+/>
```

Comment:
please remove this diagram, its a bit confusing.

Reply:
Done.